index.html

```diff
@@ -121,18 +121,51 @@ <h1>PROJECTS</h1>
       <hr class="tab-line">
 
       <!-- First tab -->
-      <div class="tab-content" data-editable="true">
-        <div class="tab-container flex-box">
-
-          <!--  -->
-          <!-- Details list of project. Dynamic using JavaScript -->
-          <!--  -->
-
-          <div class="no-data-div-details">
-            <img src="resources/imgs/error-img.png" alt="No Data available image">
-            <p> No Data Available </p>
+      <div class="project-details tab-content" data-editable="true">
+        <div class="scroll-box">
+          <div class="tab-container flex-box">
+            <!--  -->
+            <!-- Details list of project. Dynamic using JavaScript -->
+            <!--  -->
+  
+            <div class="no-data-div-details">
+              <img src="resources/imgs/error-img.png" alt="No Data available image">
+              <p> No Data Available </p>
+            </div>
+          </div>
+          <div class="resources-activity" id="resources-activity">
+            <span class="hours-spent">Hours Spent Report</span>
```

Comment:
Would be better to change to:
Total Hours Spent: value
Table

Reply:
Added label called 'Total hours spent' to the existing data section.